### PR TITLE
Feature/app top bar

### PR DESCRIPTION
**Pull request by felixheidecke**
New try ;-)

## Description
Bring the new responsive filesAppBar to the Files App

## Related Issue

- Fixes: https://github.com/owncloud/owncloud-design-system/issues/235
- Fixes: https://github.com/owncloud/phoenix/issues/1270
- Relies on: https://github.com/owncloud/owncloud-design-system/pull/356 !!

## Screenshots (if appropriate):

FireShot Capture 021 - ownCloud Phoenix - 192 168 2 82
FireShot Capture 020 - ownCloud Phoenix - 192 168 2 82
FireShot Capture 019 - ownCloud Phoenix - 192 168 2 82

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: apps/files/src/components/FilesAppBar.vue [hunk 1]
```diff
@@ -98,6 +89,15 @@ export default {
...
       return this.$gettext('Create new file…')
     },
+    _cannotCreateDialogText () {
+      let message
+
+      if (!this.canUpload) { message = 'You have no permission to upload!' }
+
+      if (!this.freeSpace) { message = 'You have not enough space left to upload!' }
+
+      return this.$gettext(message)
```
Comment (DeepDiver1975) [inline apps/files/src/components/FilesAppBar.vue]: this will not work - use 

```
this.$gettext('You have ...')

Comment (DeepDiver1975) [reply apps/files/src/components/FilesAppBar.vue]: anyhow -logic looks a bit strange - I'd do:

```
if (!this.canUpload) {
    return this.$gettext('You have no ....')
}
if (!this.freeSpace) {
    return this.$gettext('You have not ....')
}

return null

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary><p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary><p>






</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFavorites</strong> failed. Please find the screenshots inside ...</summary><p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong></strong> failed. Please find the screenshots inside ...</summary><p>








</p></details>

**Pull request merged**